aspnetcore/fundamentals/servers/index.md

```diff
@@ -16,7 +16,7 @@ An ASP.NET Core app runs with an in-process HTTP server implementation. The serv
 
 ## Kestrel
 
-Kestrel is the default web server included in ASP.NET Core project templates.
+Kestrel is the default web server specified by the ASP.NET Core project templates.
```

Comment:
@guardrex I'm changing this. The templates don't specify Kestrel. Most customers use VS and that launches IIS Express.